lib/drivers/medtronic600/NGPHistoryParser.js

```diff
@@ -1046,6 +1046,22 @@ class NGPHistoryParser {
     return _.isUndefined(programmedBolus) ? undefined : programmedBolus.timestamp;
   }
 
+  getAutoModeStartGap(autoModeStartEvent) {
+    const autoModeEventIndex = _.findIndex(this.events, event => event === autoModeStartEvent);
+
+    // Auto Mode can only last for 2 hours without a microbolus
```

Comment:
Should this search then be limited to two hours?

Reply:
Ah, no. I need to delete that comment. Turns out that there were cases where that wasn't true.